app/models/miq_event_definition_set.rb

```diff
@@ -1,6 +1,4 @@
-class MiqEventDefinitionSet < ApplicationRecord
-  acts_as_miq_set
-
+class MiqEventDefinitionSet < MiqSet
```

Comment:
do you have a different opinion about this now, since you've been reviewing [the pr that adds this table](https://github.com/ManageIQ/manageiq-schema/pull/536)?